agent/metadata_service.go

```diff
@@ -132,7 +130,7 @@ func (mds *metadataService) getCredentials(w http.ResponseWriter, r *http.Reques
 		Code:            "Success",
 		LastUpdated:     time.Now().UTC().Format(time.RFC3339),
 		Type:            "AWS-HMAC",
-		AccessKeyId:     creds.AccessKeyId,
+		AccessKeyID:     creds.AccessKeyId,
```

Comment:
Tested with the Go Playground and it does indeed change the capitalization of the returned JSON object that SDKs will consume. Please try this with a real laptop before you merge.